x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx

```diff
@@ -51,6 +51,26 @@ const i18nTexts = {
   isLoading: i18n.translate('xpack.upgradeAssistant.esDeprecations.loadingText', {
     defaultMessage: 'Loading deprecation issues…',
   }),
+  remoteClustersDetectedTitle: i18n.translate(
+    'xpack.upgradeAssistant.esDeprecations.remoteClustersDetectedTitle',
+    {
+      defaultMessage: 'Remote cluster compatibility',
+    }
+  ),
+  getRemoteClustersDetectedDescription: (remoteClustersCount: number) =>
+    i18n.translate('xpack.upgradeAssistant.esDeprecations.remoteClustersDetectedDescription', {
+      defaultMessage:
+        'You have {remoteClustersCount} {remoteClustersCount, plural, one {remote cluster} other {remote clusters}} configured. If you use cross-cluster search, note that 8.x can only search remote clusters 7.17 or later. If you use cross-cluster replication, a cluster that contains follower indices must run the same or newer version as the remote cluster.',
```

Comment:
> note that 8.x can only search remote clusters 7.17 or later.

I don't think this is accurate. Based on [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-cross-cluster-search.html#ccs-supported-configurations) (@giladgal can confirm), you can only search back one minor version.  This means an local 8.1 cluster can only search remote 8.0+ clusters. I'd change the copy as follows:

```suggestion
        'You have {remoteClustersCount} {remoteClustersCount, plural, one {remote cluster} other {remote clusters}} configured. If you use cross-cluster search, note that 8.x can only search remote clusters running the previous minor version or later. If you use cross-cluster replication, a cluster that contains follower indices must run the same or newer version as the remote cluster.',
```

Reply:
Thanks for catching this @jrodewig! Do we also need to update the "Remote cluster compatibility" note here: https://www.elastic.co/guide/en/elastic-stack/current/upgrading-elastic-stack.html?

Reply:
> Do we also need to update the "Remote cluster compatibility" note here: https://www.elastic.co/guide/en/elastic-stack/current/upgrading-elastic-stack.html?

Yes. I'll get that fixed up. Thanks, @alisonelizabeth!

Reply:
Updated the copy per your suggestion